lib/SyTest/Homeserver/Dendrite.pm

```diff
@@ -240,6 +240,19 @@ sub _get_config
                ( ! defined $ENV{'POSTGRES'} || $ENV{'POSTGRES'} == '0') ?
                "file:$self->{hs_dir}/presence.db" : $db_uri,
          },
+         threepid_database => {
+            connection_string => 
+               ( ! defined $ENV{'POSTGRES'} || $ENV{'POSTGRES'} == '0') ?
+               "file:$self->{hs_dir}/threepids.db" : $db_uri,
+         },
+         email => {
+            enabled => JSON::true,
+            from => 'synapse@localhost',
+            templates_path => '/src/res/default',
```

Comment:
this seems to be a bit of a large assumption? Who is to say that the templates are here?

Reply:
Maybe we need review from someone from Dendrite as well. I put default templates [here](https://github.com/matrix-org/dendrite/pull/1970/files#diff-37e05f9cb30ba9467db9468307a09d046072f2b517f8e7738753b2515873018d) in the mentioned PR.

Reply:
my point is that you cannot assume Dendrite is at `/src`.